frontend/src/routes/FacilityBooking/CreateBooking/index.tsx

```diff
@@ -115,9 +121,9 @@ export default function CreateBooking() {
   const FormData = watch()
```

Comment:
would this update every time the form values changes? might need to put in a useeffect for this to work or this might not be necessary since it can be used at the respective places and called when neccessary?

Reply:
[7a73fc3](https://github.com/rhdevs/rhapp/pull/637/commits/7a73fc3e18f5d6bc8ec4b124e6e4e09f0e29b664)